ambassador/ambassador/config.py

```diff
@@ -1166,6 +1167,12 @@ def generate_intermediate_config(self):
             self.envoy_clusters[cluster_key] for cluster_key in sorted(self.envoy_clusters.keys())
         ]
 
+        # Set default cors settings - default for all services without customized CORS section.
+        # Expecation is that this is directly mapped to envoy configuration, no renaming.
```

Comment:
Is this consistent with the per-`Mapping` CORS config? (I think so.)
Should we define a schema that both can validate against? (I think so, again?)